src/style.css

```
@@ -1 +1,33 @@
-/* write your styles here */
+body {
+  margin: 0 auto;
+}
+
+body div {
```

```suggestion
div {
```
why do you need body here?